routes/candidateRoutes.js

```diff
@@ -85,33 +85,49 @@ candidateRouter.post('/create', authorizedAdmin, upload.single('file'), async (r
             const command = new PutObjectCommand(params);
             const imageSaved = await s3.send(command);
             if(!imageSaved){
+                logger.error("Unable to upload image to s3");
                 return res.status(400).send({error: 'Unable to upload image'});
             }
             logger.info("Image created successfully! Details:", imageSaved);
             const candidate = new Candidate({name: name, age: age, message: message, img: `https://${BUCKET_NAME}.s3.amazonaws.com/${name}_${currentTimestamp}`});
             const candidateSaved = await candidate.save();
-            if(candidateSaved) {
+            if(candidateSaved && duplicateCandidate.length === 0) {
```

Comment:
Will simply never hit. Condition guaranteed to be false.

Reply:
Why?